poc/vidpf.py

```diff
@@ -254,14 +261,15 @@ def eval(self,
                             node,
                             onehot_proof,
                             correction_words[i],
+                            ctx,
```

Comment:
At this point I have understood what `ctx` :) but the name `ctx` is not very descriptive IMO. I understand that we have it as is to match the VDAF draft but could we have some comments in various places we use it as the name doesn't really imply what it is? (does it stand for `context`?)

Reply:
Should we enforce any constraints to the `ctx` size? The VDAF draft mentions the clients and aggregators need to agree on the string. Would it make sense to require that this is a hash of the context instead?